### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
- Fix CI

### CODE REVIEW

File: yoeo/utils/loss.py [hunk 1]
```diff
@@ -152,7 +152,20 @@ def build_targets(p, targets, model):
         t = targets * gain
         # Check if we have targets
         if nt:
-            t = t.squeeze(0)
+            # Check if we use anchor boxes
+            if na == 1:
+                # If we don't use anchor boxes we just use the first anchor box
+                t = t.squeeze(0)
+            else:
+                # Calculate ration between anchor and target box for both width and height
```
Comment (jaagut) [inline yoeo/utils/loss.py:160]:
```suggestion
                # Calculate ratio between anchor and target box for both width and height
```

File: yoeo/utils/loss.py [hunk 2]
```diff
@@ -152,7 +152,20 @@ def build_targets(p, targets, model):
         t = targets * gain
         # Check if we have targets
         if nt:
-            t = t.squeeze(0)
+            # Check if we use anchor boxes
+            if na == 1:
```
Comment (jaagut) [inline yoeo/utils/loss.py:156]: In both cases, you are doing the `t = t.squeeze(0)`, so you could simplify it.

File: yoeo/utils/loss.py [hunk 3]
```diff
@@ -152,7 +152,20 @@ def build_targets(p, targets, model):
...
+                # If we don't use anchor boxes we just use the first anchor box
+                t = t.squeeze(0)
+            else:
+                # Calculate ration between anchor and target box for both width and height
+                t = t.squeeze(0)
+                r = t[:, :, 4:6] / anchors[:, None]
+                # Select the ratios that have the highest divergence in any axis and check if the ratio is less than 4
```
Comment (jaagut) [inline yoeo/utils/loss.py:163]: Why do we care for a ratio less than four?

**Comment by jaagut** [comment]
Also, we could consider these ideas from the original PR:
https://github.com/bit-bots/YOEO/issues/44#issuecomment-1130284564

File: yoeo/utils/loss.py (hunk 2 above)
Comment (Flova) [reply yoeo/utils/loss.py:156]: You are right

**Comment by Flova** [comment]
They are nice improvements, but not straight forward, so we should maybe do it in another PR

**Comment by jaagut** [comment]
I have converted my comment mentioned earlier to a new issue #96

**Comment by Flova** [comment]
Okay 👍

**Review by jaagut — APPROVED** [review]

**Pull request merged**